contracts/scripts/DeployGatewayWithFeeInitializer.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
+pragma solidity 0.8.28;
+
+import {AgentExecutor} from "../src/AgentExecutor.sol";
+import {GatewayWithFeeInitializer} from "../src/upgrade/GatewayWithFeeInitializer.sol";
+import {ParaID} from "../src/Types.sol";
+import {Script} from "forge-std/Script.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {console} from "forge-std/console.sol";
+
+contract DeployGatewayWithFeeInitializer is Script {
```

Comment:
See my comment below about the naming style that uses the month of intended deployment. What do you think?

```suggestion
contract DeployGateway202508 is Script {
```